common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -175,4 +176,26 @@ public boolean saslServerAlwaysEncrypt() {
     return conf.getBoolean("spark.network.sasl.serverAlwaysEncrypt", false);
   }
 
+  /**
+   * The trigger for enabling AES encryption.
+   */
+  public boolean aesEncryptionEnabled() {
+    return conf.getBoolean("spark.authenticate.encryption.aes.enabled", false);
+  }
+
+  /**
+   * The implementation class for crypto cipher
+   */
+  public String aesCipherClass() {
+    return conf.get("spark.authenticate.encryption.aes.cipher.class",
+      CryptoCipherFactory.CipherProvider.OPENSSL.getClassName());
```

Comment:
This does not match the default value of the library. You should not use a default here; just use "null" as a trigger for "do not set a value", and let the library deal with the default.